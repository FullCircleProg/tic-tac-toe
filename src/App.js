import React from 'react';

export default function App () {
	return (
		<div> Play Tic-tac-toe: </div>
	)
}