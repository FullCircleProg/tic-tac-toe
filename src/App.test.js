import renderer from 'react-test-renderer';
import App from './App';

it('Renders same as before', () => {
  const component = renderer
    .create(<App />)
    .toJSON();
  expect(component).toMatchSnapshot();
});



/*************/

/*import { render, screen } from '@testing-library/react';
import App from './App';

test('renders the component', () => {
  render(<App />);
  expect(screen.getByText("<div>")).toHaveTextContent("Play Tic-tac-toe:");
});*/

/*import React from 'react';
import { shallow } from 'enzyme';
import { MyComponent } from './my.component.js';

describe(`Component: MyComponent`, () => {
  test(`MyComponent renders with default props`, () => {
    const wrapper = shallow(<MyComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});*/


