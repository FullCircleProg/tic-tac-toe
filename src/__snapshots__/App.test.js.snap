// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders same as before - basic, no crash text 1`] = `
<div>
   Play Tic-tac-toe: 
</div>
`;

exports[`Renders same as before 1`] = `
<div>
   Play Tic-tac-toe: 
</div>
`;
